packages/react-native/Libraries/Text/TextInput/RCTBackedTextInputViewProtocol.h

```diff
@@ -52,6 +52,8 @@ NS_ASSUME_NONNULL_BEGIN
 // Use `attributedText.string` instead.
 @property (nonatomic, copy, nullable) NSString *text NS_UNAVAILABLE;
 
+@property (nonatomic, assign) BOOL disableKeyboardShortcuts;
```

Comment:
If its already defined on the protocol do we need to redefine it on the BaseTextInputView?

Reply:
Done - 7608350f874f2a5e904d1fffd748122a8d691526.